### PR TITLE
好室友机器人升级为企业微信, 结合使用ChatGPT

### PR DESCRIPTION
**Pull request by aishiqi**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/jekyll/blob/main/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

File: jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+
+好室友机器人由于之前个人微信号不稳定，不工作有一段时间了，最近我们和Huan，Jiarui聊天中了解到使用企业微信可以降低这个风险，于是我们将好室友机器人升级成了企业号。
+企业号需要在国内挂靠国内实体企业，开通后，将群主转让给企业微信号，可以将普通微信群转换成企业群。
+
+Setup：
+
+```Text
+我们使用句子Fork的版本，例如
+"@juzi/wechaty": "^1.0.52",
```
Comment (huan) [inline jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md:20]: Can we use the official Wechaty npm for this project?

**Comment by huan** [comment]
Please make sure the following requirements have been met for your PR before we can continue reviewing it:

2. The PR has passed all the CI tests and turned _green_. (you can execute all tests locally by running `npm test`)

**Comment by aishiqi** [comment]
> Please make sure the following requirements have been met for your PR before we can continue reviewing it:
> 
> 2. The PR has passed all the CI tests and turned _green_. (you can execute all tests locally by running `npm test`)

Done, passed the test.

File: jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md (hunk 1 above)
Comment (aishiqi) [reply jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md]: Sure, I tested, both of them works.
The Juzi version may have some extra features.
For example, it support those extra events:
```Text
post-comment
post-tap
room-announce
tag
tag-group
```

I've just updated the blog to explain it.

File: jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+tag-group
+```
+
+更多详情请看 [Diff](https://github.com/wechaty/puppet/compare/main...juzibot:wechaty-puppet:main#diff-6125e4db0556b5355b2c93d63632b29f6192003c9b3e36835d0c9f75f961fe9dR17)
+
+如果需要这些功能的话，需要以下改动：
+
+```Text
+package.json:
+"@juzi/wechaty": "^1.0.52",
```
Comment (huan) [inline jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md:63]: Can we use the community npm package instead of an unofficial one?

**Review by huan — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md (hunk 2 above)
Comment (aishiqi) [reply jekyll/_posts/2023-05-21-haoshiyou-bot-upgrade-to-wecom.md]: Sure, Removed the unofficial wechaty related content.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> 

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message. (See also: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16))

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/jekyll/tree/main/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add @lijiarui wechat: _ruirui_0914_ and send her this pr link. She will invite you into Wechaty Contributor Room

Cheers!

**Pull request merged**